src/main/java/com/puppycrawl/tools/checkstyle/TreeWalker.java

```diff
@@ -95,6 +98,16 @@ public void setModuleFactory(ModuleFactory moduleFactory) {
         this.moduleFactory = moduleFactory;
     }
 
+    /**
+     * Setter to control whether to skip files with Java parsing errors.
+     *
+     *  @param skipFileOnJavaParseException whether to always check for a trailing comma.
+     *  @since 10.15.1
```

Comment:
10.16.0 , it will be new feature, that people awaits alot.

Reply:
done